frontend/model/state.js

```diff
@@ -108,37 +127,59 @@ const mutations = {
     Vue.delete(state.chatRoomScrollPosition, chatRoomId)
   },
   setChatRoomReadUntil (state, { chatRoomId, messageHash, createdDate }) {
-    const prevMentions = state.chatRoomUnread[chatRoomId].mentions
-    const prevOthers = state.chatRoomUnread[chatRoomId].others
-
+    // eslint-disable-next-line no-lone-blocks
+    {
+      // TODO: need to remove this block after we release 0.2.*
+      if (!state.chatRoomUnread[chatRoomId].messages) {
+        Vue.set(
+          state.chatRoomUnread[chatRoomId],
+          'messages',
+          [].concat(
+            (state.chatRoomUnread[chatRoomId].mentions || []).map(m => Object.assign({ type: MESSAGE_TYPES.TEXT }, m)),
+            (state.chatRoomUnread[chatRoomId].others || []).map(o => Object.assign({ type: MESSAGE_TYPES.INTERACTIVE }, o))
+          )
+        )
+      }
```

Comment:
I don't understand the purpose of this block... shouldn't this have already been taken care of by `'state/vuex/postUpgradeVerification'`?